examples/arquillian-jpa/pom.xml

```diff
@@ -28,10 +28,22 @@ language governing permissions and limitations under the License. -->
         <artifactId>maven-compiler-plugin</artifactId>
         <version>3.5.1</version>
         <configuration>
-          <source>1.8</source>
-          <target>1.8</target>
+          <source>11</source>
+          <target>11</target>
         </configuration>
       </plugin>
+      <plugin>
+       <groupId>org.apache.maven.plugins</groupId>
+       <artifactId>maven-surefire-plugin</artifactId>
+       <version>2.22.2</version>
+       <dependencies>
+                <dependency>
+                    <groupId>org.junit.jupiter</groupId>
+                    <artifactId>junit-jupiter-engine</artifactId>
+                    <version>5.7.0</version>
```

Comment:
Any reason for `5.7.0` vs `5.8.2` ?

Reply:
There is no particular reason the use of 5.7.0 so I upgraded it to 5.8.2